symplyphysics/laws/astronomy/relativistic/speed_of_rocket_depends_on_mass_and_impulse.py

```diff
@@ -0,0 +1,50 @@
+from sympy import (Eq, solve)
```

Comment:
CI is failing due to duplicate law names. Add something to this law name to distinguish them.